app/models/concerns/generate_csv.rb

```diff
@@ -0,0 +1,26 @@
+module GenerateCsv
+  extend ActiveSupport::Concern
+  require 'csv'
+
+  class_methods do
+    def to_csv
+      CSV.generate(col_sep: ';') do |csv|
+        csv << column_names_for_export
+        order(created_at: :desc).each do |item|
```

Comment:
@MartGS let's remove `order(created_at: :desc)` from this method, so we will use order of the collection that was applied above of it.